packages/design-system-tokens/src/index.ts

```diff
@@ -53,7 +65,12 @@ const tokenExporter = (inputType: string, exportType: string): number => {
   if (!INPUT_TYPES.includes(inputType)) help(`valid import types are: ${INPUT_TYPES}`);
   if (!EXPORT_TYPES.includes(exportType)) help(`valid export types are: ${EXPORT_TYPES}`);
 
-  const res = tokenExporter(inputType, exportType);
+  let res;
+  if (exportType === 'css-vars') {
+    res = cssVarExporter(inputType);
+  } else {
+    res = tokenExporter(inputType, exportType);
+  }
```

Comment:
Since there is already some conditional logic about the export type in the `tokenExporter` function, is there a way to marry the two functions again? It actually looks like the logic inside them is exactly the same? Same `outputPath` being calculated